### ISSUE TITLE
COMPLETE - Lighthouse Tool Accessibility Test - contrast

### ISSUE THREAD

**Issue by gacetta** [roundhousedesigns/rise-frontend#57]
Uncertain if the current color scheme will pass all accessibility requirements.  Run lighthouse tool accessibility tests and share results with design team.

**Comment by gacetta** [comment]
Ran Lighthouse Tests on all pages and got some results!  I’ll post .html files of all of them here and then maybe open some GitHub Issues to investigate further:

P: performance, A: accessibility, BP: best practices, SEO: search engine optimization
Scores (percentages) for each page as Desktop in incognito mode using actual URL (not dev environment)

**LoginView**: P - 25, A - 98, BP - 100, SEO - 85
**RegisterView**: P - 13, A - 96, BP - 100, SEO - 85
**DashboardView**: P - 64, A - 98, BP - 100, SEO - 82
**ProfileView**: P - 50, A - 98, BP - 100, SEO - 82
**EditProfileView**: P - 51, A - 84, BP - 100, SEO - 73
**Settings / ChangePasswordView / ChangeProfileSlugView**: P - 51, A - 89, BP - 100, SEO - 82
**Help / ContentView**: P - 51, A - 89, BP - 100, SEO - 82
**SearchResultsView**: P - 52, A - 96, BP - 100, SEO - 82
**LostPasswordView**: P - 71, A - 90, BP - 100, SEO - 82

The color contrast issue wasn’t a thing at all!  You surpass the 4.5:1 ratio (I ran my own check and orange/black was 6.6:1) so everything is good there!

**Comment by gacetta** [comment]
Will read reports for further issues to look into.

This issue is ready to close.

**Comment by gaswirth** [comment]
Awesome! Thanks for running these. Glad to see some good Accessibility and BP numbers. I'll add an Issue about the performance scores and will reference this thread. Thanks @gacetta !
